docs-source/spring/content/development/project.md

```diff
@@ -100,13 +106,11 @@ management:
       application: ${spring.application.name}
 ```
 
-The variables in this configuration are automatically injected to your deployment
-and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application. 
+The variables in this configuration are automatically injected to your deployment and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application.
 
-#### Data Sources
+### Data Sources
 
-If your application uses a data source, then add the following configuration.  Note that this
-example shows Java Persistence API (JPA), if you are using JDBC you should use the appropriate configuration.
+If your application uses a data source, then add the following configuration.  Note that this example shows Java Persistence API (JPA), if you are using JDBC you should use the appropriate configuration.
```

Comment:
Make the second sentence two sentences. For example, "Note that this example shows Java Persistence API (JPA). If you are using JDBC, use the appropriate configuration."